skietbaan_fe/src/components/Login.js

```diff
@@ -114,64 +157,84 @@ class Login extends Component {
     }
   }
 
+  goToRegister() {
+    window.location = "/Register-page"
+  }
+
   render() {
     document.addEventListener('DOMContentLoaded', () => {
       this.toggleNavbar();
-   }, false);
-    if(getCookie("token")){
+    }, false);
+    if (getCookie("token")) {
       window.location = "/home";
     }
 
     return (
-      <div className="Page-content">
-      <div className = "welcome-header"><label className="welcome-label">Welcome back to skietbaan</label></div>
-      <div className="header-container">
-      <label className = "header-label">Login</label>
-      </div>
+      <div className="page-content-login">
```

Comment:
inconsistent class names. Take note of earlier comments. Choose between camelCase or dash-notation but not CapitalCase

Reply:
done